frontend/hooks/useShouldRedirect.js

```diff
@@ -1,3 +1,5 @@
+import { USER_REGISTRATION_REQUIRED_FORM_FIELDS } from '../constants';
```

Comment:
When I try to run the application, I get a build error:

```
Module not found: Can't resolve '../constants'
> 1 | import { USER_REGISTRATION_REQUIRED_FORM_FIELDS } from '../constants';
```

Was this file not committed mistakenly?

Reply:
Thanks for catching this. It is now included.